integration

```diff
@@ -1492,6 +1492,7 @@
   "zeronounours/HA-custom-component-energy-meter",
   "zhbjsh/homeassistant-ssh",
   "zigul/HomeAssistant-CEZdistribuce",
+  "zollak/homeassistant-syslog-receiver",
```

Comment:
<https://github.com/zollak/homeassistant-syslog-receiver/blob/v1.0.7/hacs.json#L4> ALL should not be used as country.
This needs to be addressed, and then released, before this can be included.